pom.xml

```diff
@@ -101,6 +101,11 @@
   </distributionManagement>
 
   <dependencies>
+    <dependency>
+     <groupId>io.minio</groupId>
+     <artifactId>minio</artifactId>
+     <version>5.0.0</version>
+    </dependency>
```

Comment:
https://github.com/minio/minio-java/blob/5.0.0/README.md indicates that this requires Java 1.8+.

Reply:
Discussed today, we'll go with jdk8 for now and review later if necessary.